api/src/main/java/io/opentelemetry/distributedcontext/DistributedContext.java

```diff
@@ -33,12 +33,12 @@
 @Immutable
 public interface DistributedContext {
   /**
-   * Returns an iterator over the entries in this {@code DistributedContext}.
+   * Returns the entries in this {@code DistributedContext}. Order of entries is not guaranteed.
    *
-   * @return an iterator over the entries in this {@code DistributedContext}.
+   * @return the entries in this {@code DistributedContext}.
```

Comment:
You could clarify that an unmodifiable collection will be returned, although `@Immutable` on the class should be sufficient.

Reply:
Make sense, updated.